.github/workflows/bun-types-tests.yml

```diff
@@ -36,5 +36,20 @@ jobs:
       - name: Generate package
         run: bun run build
 
+      - name: Shim Prettier
+        run: |
+          prettier_dir=$(mktemp -d)
+          cat > "$prettier_dir/prettier" <<EOF
+          #!/bin/sh
+          bun prettier "$@"
+          EOF
+          chmod +x "$prettier_dir/prettier"
```

Comment:
This could be an actual file in the repository, but it'd want to be a file in a directory all its own so that it could be added to the PATH w/o adding anything else to the path, and it'd want to be written in such a way that people understand that it's *only* for use here.